packages/client/src/pages/GamePage/GamePage.tsx

```diff
@@ -1,13 +1,279 @@
-import { Text } from '@gravity-ui/uikit';
+import { useEffect, useRef } from 'react';
 
-import { Container, Page } from 'src/components';
+import { useNavigate } from 'react-router-dom';
+
+import {
+  GpConst,
+  SPRITES,
+  cubeDatas,
+  defaultHeight,
+  defaultWidth,
+  shiftHorizontal,
+  shiftVertical,
+} from './GamePage.constants';
+
+import { TFigure, TRectClickHandler } from './GamePage.types';
+
+import { GpUtils } from './GamePage.utils';
+
+import { PAGE_ROUTES } from '../../utils/constants';
+
+import { loadSprites } from '../../utils/loadSprites';
+
```

Comment:
Будет классно везде использовать абсолютные импорты. Это считается [хорошей практикой](https://medium.com/@pushplaybang/absolutely-dont-use-relative-paths-imports-in-your-vite-react-project-c8593f93bbea)

```suggestion
import { PAGE_ROUTES } from 'src/utils/constants';

import { loadSprites } from 'src/utils/loadSprites';

```